File: Symfony/src/BGR/Serrano/ProductoBundle/Entity/Remanente.php
<?php

namespace BGR\Serrano\ProductoBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use JMS\Serializer\Annotation\Type;
/**
 * Remanente
 *
 * @ORM\Table()
 * @ORM\Entity(repositoryClass="BGR\Serrano\ProductoBundle\Entity\RemanenteRepository")
 */
class Remanente
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @ORM\ManyToOne(targetEntity="Producto") 
     * @ORM\JoinColumn(name="producto_id", referencedColumnName="id")
     * @Type("BGR\Serrano\ProductoBundle\Entity\Producto")
     *
     */
    protected $producto;

    /**
     * @var float
     *
     * @ORM\Column(name="cantidad", type="decimal")
 	  * @Type("double")
     */
    private $cantidad;
    
   /**
     * @ORM\ManyToOne(targetEntity="UnidadDeMedida") 
     * @ORM\JoinColumn(name="unidadDeMedida_id", referencedColumnName="id")
     * @Type("BGR\Serrano\ProductoBundle\Entity\UnidadDeMedida")
     */
    protected $unidad_de_medida;


    /**
     * Get id
     *
     * @return integer 
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set producto
     *
     * @param Producto $producto
     * @return Remanente
     */
    public function setProducto($producto)
    {
        $this->producto = $producto;
    
        return $this;
    }

    /**
     * Get producto
     *
     * @return Producto 
     */
    public function getProducto()
    {
        return $this->producto;
    }

    /**
     * Set cantidad
     *
     * @param float $cantidad
     * @return Remanente
     */
    public function setCantidad($cantidad)
    {
        $this->cantidad = $cantidad;
    
        return $this;
    }

    /**
     * Get cantidad
     *
     * @return float 
     */
    public function getCantidad()
    {
        return $this->cantidad;
    }
    
    /**
     * Set UnidadDeMedida
     *
     * @param UnidadDeMedida $unidad_de_medida
     * @return Remanente
     */
    public function setUnidadDeMedida(UnidadDeMedida $unidad_de_medida)
    {
        $this->unidad_de_medida = $unidad_de_medida;
    
        return $this;
    }

    /**
     * Get unidad_de_medida
     *
     * @return UnidadDeMedida 
     */
    public function getUnidadDeMedida()
    {
        return $this->unidad_de_medida;
    }
}

File: BGR/BGR-Data/src/main/java/ar/com/bgr/serrano/model/Remanente.java
package ar.com.bgr.serrano.model;

import javax.persistence.CascadeType;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.Table;

@Entity(name="Remanente")
@Table(name="Remanente")
public class Remanente {

    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
	@Id
	@GeneratedValue
    private Integer id;

    /**
     * @ORM\ManyToOne(targetEntity="Producto") 
     * @ORM\JoinColumn(name="producto_id", referencedColumnName="id")
     * @Type("BGR\Serrano\ProductoBundle\Entity\Producto")
     *
     */
	@ManyToOne(cascade=CascadeType.ALL)
	@JoinColumn(name="producto_id")
    private Producto producto;

    /**
     * @var float
     *
     * @ORM\Column(name="cantidad", type="decimal")
 	  * @Type("double")
     */
	@Column(name="cantidad")
    private Double cantidad;
    
   /**
     * @ORM\ManyToOne(targetEntity="UnidadDeMedida") 
     * @ORM\JoinColumn(name="unidadDeMedida_id", referencedColumnName="id")
     * @Type("BGR\Serrano\ProductoBundle\Entity\UnidadDeMedida")
     */
	@ManyToOne(cascade=CascadeType.ALL)
	@JoinColumn(name="unidadDeMedida_id")
    protected UnidadDeMedida unidad_de_medida;

	/**
	 * @return the id
	 */
	public Integer getId() {
		return id;
	}
	
	/**
	 * @param id the id to set
	 */
	public void setId(Integer id) {
		this.id = id;
	}
	
	/**
	 * @return the producto
	 */
	public Producto getProducto() {
		return producto;
	}
	
	/**
	 * @param producto the producto to set
	 */
	public void setProducto(Producto producto) {
		this.producto = producto;
	}
	
	/**
	 * @return the cantidad
	 */
	public Double getCantidad() {
		return cantidad;
	}
	
	/**
	 * @param cantidad the cantidad to set
	 */
	public void setCantidad(Double cantidad) {
		this.cantidad = cantidad;
	}
	
	/**
	 * @return the unidad_de_medida
	 */
	public UnidadDeMedida getUnidad_de_medida() {
		return unidad_de_medida;
	}
	
	/**
	 * @param unidad_de_medida the unidad_de_medida to set
	 */
	public void setUnidad_de_medida(UnidadDeMedida unidad_de_medida) {
		this.unidad_de_medida = unidad_de_medida;
	}
	
	
}
